oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/DocumentNodeStoreBuilder.java

```diff
@@ -324,6 +346,11 @@ public T setNoChildOrderCleanupFeature(@Nullable Feature noChildOrderCleanupFeat
         return thisBuilder();
     }
 
+    public T setDocStoreFullGCFeature(@Nullable Feature docStoreFullGC) {
+        this.docStoreFullGCFeature = docStoreFullGC;
+        return thisBuilder();
+    }
+
```

Comment:
That's out of place. I'd suggest moving it down a few lines to where it belongs.

Reply:
Addressed in https://github.com/apache/jackrabbit-oak/pull/1482/commits/6b699d245d42d4272e5d7ba4e833d195e7586706